embassy-stm32/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 #![no_std]
 #![cfg_attr(
     feature = "nightly",
-    feature(type_alias_impl_trait, async_fn_in_trait, impl_trait_projections)
+    feature(type_alias_impl_trait, async_fn_in_trait, impl_trait_projections, int_roundings)
```

Comment:
We shouldn't add new nightly features. The already present ones are absolutely required for async traits, and likely to be stabilized soon. The goal is for the whole of Embassy to work on stable when that happens.